milmove-atlantis/Dockerfile

```
@@ -18,10 +18,9 @@ RUN set -ex \
 # Install Python3 for Lambdas
 RUN apk add --no-cache python3
 
-RUN git config --global --add safe.directory "*"
-
 LABEL name="atlantis"
 
 ENV DEFAULT_TERRAFORM_VERSION=1.1.2
+ENV GIT_CEILING_DIRECTORIES="/home/atlantis/.atlantis/repos/transcom/transcom-infrasec-com"
```

I'm not sure if the paths are accurate here. But if we need to add more, the suggestion below works as the variable is `:` colon-delimited like the `PATH` variable in Linux.

```suggestion
ENV GIT_CEILING_DIRECTORIES="/home/atlantis/.atlantis/repos/transcom/transcom-infrasec-com:/home/atlantis/.atlantis/repos/transcom/transcom-infrasec-gov"
```